front/pages/api/w/[wId]/groups/[gId]/index.ts

```diff
@@ -71,6 +71,15 @@ async function handler(
           },
         });
       }
+      if (groupRes.value.kind === "system") {
+        return apiError(req, res, {
+          status_code: 400,
```

Comment:
404 no?

Reply:
Agreed 400 not great, I've put a 403 instead!